Mathlib/Control/ULiftable.lean

```diff
@@ -110,6 +110,9 @@ open ULift
 instance : ULiftable id id where
   congr F := F
 
+instance : ULiftable Id Id where
+  congr F := F
```

Comment:
I thought we weren't supposed to use `id` anywhere for monads; should we only have the second of these?